Add an option to specify activeClass by default to multiple elements (#48)
CoherentLabs/GameUIComponents: components/router/README.md

`<gameface-route activeClass="myActiveStyle">`

> You can apply the activeClass to multiple `<gameface-route>` elements by adding it to a container:

Reviewer: ```suggestion
You can apply the `activeClass` to multiple `<gameface-route>` elements by adding it to a container:
```

Author: Changed.

```
<div activeClass="myActiveStyle">
    <gameface-route to="/">Home</gameface-route>
    <gameface-route to="/play">Play</gameface-route>
    <gameface-route to="/settings">Settings</gameface-route>
</div>
```

> If you have both activeClass on the wrapper an on some individual elements, those routes that don't have own activeClass will inherit it from the parent element and those that have will use their own:

Reviewer: ```suggestion
If you have both `activeClass` on the wrapper an on some individual elements, those routes that don't have own `activeClass` will inherit it from the parent element and those that have will use their own:
```

Author: Changed.

```
<div activeClass="green">
    <gameface-route to="/">Home</gameface-route> // will be green
    <gameface-route activeClass="red" to="/play">Play</gameface-route> // will be red
    <gameface-route to="/settings">Settings</gameface-route> // will be green
</div>
```

> If you don't add activeClass the router element will use a default light blue color.

Reviewer: Maybe this is more valid explanation?

```suggestion
If you don't add `activeClass` to the wrapper div and the route element then the default light blue color will be used.
```

Author: Changed.
